scripts/clang-format.sh

```diff
@@ -301,8 +301,7 @@ function RequireProgram {
 
 # Make sure we are running from the top-level git directory.
 # Same approach as for setup-decoder.sh. Good enough.
-# We could probably use git rev-parse --show-toplevel to do so, as long as we
-# handle the libhtp subfolder correctly.
+# We could probably use git rev-parse --show-toplevel to do so
```

Comment:
Now that we get rid of libhtp C, is that comment possible to do ?

Reply:
Possibly. `git rev-parse --show-toplevel` just returns the top of the source directory, so we could test `[ -e  $(git rev-parse --show-toplevel)/src/suricata.c ]` and then change to that directory. 

But I thought at the time that this was outside the scope of this PR? I can do it if you would like, but I was thinking it would be something that could be done after.

Reply:
Low priority indeed